htsinfer/get_read_orientation.py

```diff
@@ -416,6 +101,7 @@ def process_alignments(
             if star_dirs[0].name == "file_1":
                 results.file_1 = self.process_single(sam=paths[0])
         elif len(star_dirs) == 2:
+            print("Getting there")
```

Comment:
I think this is a leftover from testing. If not, please change into a logging message

Reply:
opsie 😆